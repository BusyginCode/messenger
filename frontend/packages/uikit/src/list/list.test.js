import React from 'react'
import { shallow } from 'enzyme'
import List from './list'
import Text from '../text'

describe('uikit/List', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <List items={[{ value: <Text>Custom config Icon 1</Text> }, { value: <Text>Custom config Icon 2</Text> }]} />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
