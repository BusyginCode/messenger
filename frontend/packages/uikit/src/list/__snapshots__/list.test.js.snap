// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`uikit/List should render correctly 1`] = `
<Box
  component="div"
  direction="column"
>
  <ListItem
    iconMargin="m"
    key="0"
    marginBottom="s"
  >
    <Text
      align="left"
      component="p"
      size="m"
      typeface="base"
    >
      Custom config Icon 1
    </Text>
  </ListItem>
  <ListItem
    iconMargin="m"
    key="1"
    marginBottom="s"
  >
    <Text
      align="left"
      component="p"
      size="m"
      typeface="base"
    >
      Custom config Icon 2
    </Text>
  </ListItem>
</Box>
`;
